packages/plugins/content/slate/src/Component/index.js

```diff
@@ -37,6 +35,12 @@ const onBlur = (_event, _data, state) => state
 
 export type Props = ContentPluginProps<{ editorState: Object }>
 
+const theme = createMuiTheme({
```

Comment:
Would it maybe make more sense, that since the `ThemeProvider` is exported by `ory-editor-ui`, to have that handled there? Either by exporting the `darkPalette` in the ui lib or by having a `DarkThemeProvider`?

Reply:
We could, this approach seemed more flexible if further theme customisation is required.